src/components/CommonLayout/CommonLayout.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Header from './Header';
+import Header from '../Header/Header';
```

Comment:
same thing as previous comment. I think its better to change the name of the second Header.. but we can ask our mentor what is the best practice.